tests/modules/nf-core/universc/test.yml.backup

```diff
@@ -0,0 +1,150 @@
+## nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml universc
```

Comment:
What's the point of this backup file?

Reply:
Sorry I committed this by mistake. I used it to compare the files before and after running `nf-core modules create-test-yml universc` and `prettier tests/modules/nf-core/universc/test.yml`. I'll remove it.